app/activeplan/homepage/contactArrangements/post.controller.js

```diff
@@ -0,0 +1,43 @@
+const { body } = require('express-validator')
+const { logger } = require('../../../../common/logging/logger')
+const { setSentencePlanComment } = require('../../../../common/data/sentencePlanningApi')
+const { getContactArrangements } = require('./get.controller')
+
+const validationRules = () => {
+  return [
+    body('contactArrangements')
+      .isLength({ min: 1 })
+      .withMessage('Enter contact arrangements'),
+    body('contactArrangements')
+      .trim()
+      .escape(),
+  ]
+}
+
+const postContactArrangements = async (req, res) => {
+  const {
+    path,
+    errors,
+    params: { planId },
+    headers: { 'x-auth-token': token },
+    body: { contactArrangements },
+  } = req
+  if (errors) {
+    return getContactArrangements(req, res)
+  }
+  try {
+    const comment = [
+      {
+        comment: contactArrangements,
+        commentType: 'LIAISON_ARRANGEMENTS',
```

Comment:
Can you raise a ticket to make some constants please?

Reply:
Done - https://dsdmoj.atlassian.net/browse/SP-560